module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -19,268 +19,133 @@
-#include "message/platform/RawSensors.hpp"
+#include "Kinematics.hpp"
+#include "Odometry.hpp"
+#include "RawSensors.hpp"
 
-#include "utility/actuation/ForwardKinematics.hpp"
-#include "utility/input/LimbID.hpp"
-#include "utility/input/ServoID.hpp"
-#include "utility/math/euler.hpp"
-#include "utility/nusight/NUhelpers.hpp"
-#include "utility/platform/RawSensors.hpp"
-#include "utility/support/yaml_expression.hpp"
+#include "extension/Configuration.hpp"
 
 namespace module::input {
 
     using extension::Configuration;
 
-    using message::actuation::BodySide;
-    using message::actuation::KinematicsModel;
-    using message::input::Sensors;
-    using message::platform::ButtonLeftDown;
-    using message::platform::ButtonLeftUp;
-    using message::platform::ButtonMiddleDown;
-    using message::platform::ButtonMiddleUp;
-    using message::platform::RawSensors;
-
-    using utility::actuation::kinematics::calculateAllPositions;
-    using utility::actuation::kinematics::calculateCentreOfMass;
-    using utility::actuation::kinematics::calculateInertialTensor;
-    using utility::input::ServoID;
-    using utility::math::euler::MatrixToEulerIntrinsic;
-    using utility::nusight::graph;
-    using utility::support::Expression;
-
-    std::string makeErrorString(const std::string& src, uint errorCode) {
-        std::stringstream s;
-
-        s << "Error on ";
-        s << src;
-        s << ":";
-
-
-        if ((errorCode & RawSensors::Error::INPUT_VOLTAGE) != 0u) {
-            s << " Input Voltage ";
-        }
-        if ((errorCode & RawSensors::Error::ANGLE_LIMIT) != 0u) {
-            s << " Angle Limit ";
-        }
-        if ((errorCode & RawSensors::Error::OVERHEATING) != 0u) {
-            s << " Overheating ";
-        }
-        if ((errorCode & RawSensors::Error::OVERLOAD) != 0u) {
-            s << " Overloaded ";
-        }
-        if ((errorCode & RawSensors::Error::INSTRUCTION) != 0u) {
-            s << " Bad Instruction ";
-        }
-        if ((errorCode & RawSensors::Error::CORRUPT_DATA) != 0u) {
-            s << " Corrupt Data ";
-        }
-        if ((errorCode & RawSensors::Error::TIMEOUT) != 0u) {
-            s << " Timeout ";
-        }
-
-        return s.str();
-    }
+    SensorFilter::SensorFilter(std::unique_ptr<NUClear::Environment> environment) : Reactor(std::move(environment)) {
 
-    SensorFilter::SensorFilter(std::unique_ptr<NUClear::Environment> environment)
-        : Reactor(std::move(environment)), theta(Eigen::Vector3d::Zero()) {
+        on<Configuration>("SensorFilter.yaml").then([this](const Configuration& config) {
+            log_level = config["log_level"].as<NUClear::LogLevel>();
 
-        on<Configuration>("SensorFilter.yaml").then([this](const Configuration& cfg) {
-            log_level = cfg["log_level"].as<NUClear::LogLevel>();
+            // **************************************** Button config ****************************************
```

Comment:
low key would be in favour of moving config to it's own cpp file in a function. it takes up like 1/3 of the file :vomiting_face: